lib/carrierwave-aws.rb

```diff
@@ -43,7 +44,33 @@ def self.normalized_acl(acl)
     normalized
   end
 
+  def self.aws_signer(value = nil)
+    self.aws_signer = value if value
```

Comment:
I'm iffy about this, but I suppose this is for compatibility and was only added as a way to bypass automatic Proc expansion.